Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -577,3 +588,9 @@ struct PaymentSheetTestPlayground_Previews: PreviewProvider {
         PaymentSheetTestPlayground(settings: .defaultValues())
     }
 }
+
+extension View {
+    func hidden(_ shouldHide: Bool) -> some View {
+        opacity(shouldHide ? 0 : 1)
+    }
+}
```

Comment:
iiuc this approach leaves an empty space when `shouldHide` is true, is that right?

If so, i think we want to hide by conditionally including the group itself.  Like
```
if playgroundController.settings.uiStyle == .embedded {
  Group { ... }
}
```

Reply:
oh yeah better nice